src/hotspot/share/services/diagnosticCommand.hpp

```diff
@@ -158,6 +158,24 @@ class JVMTIAgentLoadDCmd : public DCmdWithParser {
   static const char* impact() { return "Low"; }
   virtual void execute(DCmdSource source, TRAPS);
 };
+
+// SapMachine 2024-12-05
+#if defined(WITH_SAP_JMC_AGENT)
+class JVMTIJmcAgentLoadDCmd : public DCmdWithParser {
+protected:
+  DCmdArgument<char*> _option;
+public:
+  JVMTIJmcAgentLoadDCmd(outputStream* output, bool heap);
+  static int num_arguments() { return 1; }
+  static const char* name() { return "JVMTI.jmc_agent_load"; }
+  static const char* description() {
+    return "Load the JCM agent.";
```

Comment:
typo - JMC agent  ;  and maybe better say SAP jmc agent of something more specific ?